### PR TITLE
Separate options for footnotes and cross references

### LINKED ISSUE #2496: Separate configuration options for showing footnotes and cross-references
Upon opening the settings menu for a workspace, it's currently only possible to show or hide *both*, footnotes *and* cross-references. It would be great if these could be separate configuration options since I usually want to display footnotes but hide cross-references for better readability.

### PR DESCRIPTION
**Pull request by agrogers**
Resolves #2496.
I am not sure where the icon comes from. And i am not sure what icon to use.

image

### CODE REVIEW

File: app/src/main/java/net/bible/service/db/DatabaseContainer.kt [hunk 1]
```diff
@@ -1050,6 +1050,17 @@ private val MIGRATION_64_65_sync_group = object : Migration(64, 65) {
         }
     }
 }
+private val MIGRATION_65_66_add_Xrefs_option = object : Migration(65, 66) {
+    override fun doMigrate(db: SupportSQLiteDatabase) {
+        db.apply {
+            val colDefs = listOf<String>("`text_display_settings_showXrefs` INTEGER DEFAULT NULL")
+            colDefs.forEach {
+                execSQL("ALTER TABLE `Workspace` ADD COLUMN $it")
+                execSQL("ALTER TABLE `PageManager` ADD COLUMN $it")
```
Comment (tuomas2) [inline app/src/main/java/net/bible/service/db/DatabaseContainer.kt:1059]: TODO: need to set value of xrefs option initially to be the same as footnote option value currently is.

**Comment by tuomas2** [comment]
Tip how to figure out how icons are defined:
In Android Studio find FOOTNOTES type in WorkspaceEntities.kt -> Types.

Click FOOTNOTES with right button -> "find usages".
Icon definitions are found then in OptionsMenuItems.kt.

File: app/src/main/java/net/bible/service/db/DatabaseContainer.kt (hunk 1 above)
Comment (agrogers) [reply app/src/main/java/net/bible/service/db/DatabaseContainer.kt:1059]: This is done now.

**Comment by agrogers** [comment]
> Tip how to figure out how icons are defined: In Android Studio find FOOTNOTES type in WorkspaceEntities.kt -> Types.
> 
> Click FOOTNOTES with right button -> "find usages". Icon definitions are found then in OptionsMenuItems.kt.

Icon added.

**Comment by tuomas2** [comment]
Thanks for doing this!

**Pull request merged**